doc/help/pypeit_show_1dspec.rst

```diff
@@ -19,4 +19,5 @@
       --extract EXTRACT  Extraction method. Default is OPT. ['BOX', 'OPT'] (default:
                          OPT)
       --flux             Show fluxed spectrum? (default: False)
+      -m, --masked       Only show unmasked data. (default: False)
```

Comment:
I would think the default was to plot gpm*flux?

Reply:
It's not currently the default.  Should I make it so?

Reply:
Just saw you're comment below.  I'll make masking the default, and change the sense of the option.